src/pudl/metadata/fields.py

```diff
@@ -364,9 +364,9 @@
         "description": "Account balance at end of year.",
         "unit": "USD",
     },
-    "ferc_account_description": {
-        "type": "string",
-    },
+    "energy_source_mwh": {"type": "number", "unit": "MWh"},
```

Comment:
I honestly don't know what this columns columns be named. i am tempted to just name it `generation_mwh` or `energy_mwh`. both the source and disposition table will have a `mwh` column. Based on our general naming conventions, I think these two columns should have the same name.

Reply:
I don't think we really know what the right distinction is here until we see other tables. I'm definitely inclined to go with `energy_mwh` because we've been overly specific in the past and had to go back and change a bunch of names because of it.

the main thing I can understand about sources vs dispositions is that the totals of each should equal each other, even though that is not a calculated value. So I don't think dispositions should actually be reported as negative values.

![image](https://user-images.githubusercontent.com/25487998/204646185-1196070a-1514-488a-81cc-1e65bc76da83.png)